bodhi/server/models.py

```diff
@@ -1506,6 +1506,21 @@ def get_testcase_karma(self, testcase):
                         bad += 1
         return bad * -1, good
 
+    def beautify_title(self, amp=False):
```

Comment:
Let's give this method a docblock explaining what it does, documenting the ```amp``` parameter, its type, and its default, and also documenting that it returns a basestring.